src/Map.container.ts

```diff
@@ -38,13 +40,18 @@ const getGeneratorConfigWithData = createSelector(
   }
 )
 
+const getLoading = createSelector([getLoaders], (loaders: Loader[]): boolean => {
```

Comment:
Maybe the types inference is not working well but is not necessary to include the type in the function so 
```suggestion
const getLoading = createSelector([getLoaders], (loaders): boolean => {
```
should be enough to know the loaders is an array of Loaders.